src/jitlayers.cpp

```diff
@@ -808,13 +808,21 @@ JL_JITSymbol JuliaOJIT::findUnmangledSymbol(StringRef Name)
 uint64_t JuliaOJIT::getGlobalValueAddress(StringRef Name)
 {
     auto addr = findSymbol(getMangledName(Name), false);
-    return addr ? cantFail(addr.getAddress()) : 0;
+    if (Error Err = addr.takeError()) {
+        handleAllErrors(std::move(Err), [](ErrorInfoBase &EIB) {}); // handle Error for assertions
+        return 0;
+    }
```

Comment:
yeah, llvm has an alias for this, which is the same code, but can be slightly more concise:
```suggestion
    if (!addr) {
        consumeError(addr.takeError());
        return 0;
    }
```